src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestUrlBuilder.java

```diff
@@ -265,6 +265,7 @@
 
   @Test(groups = {"TestUrlBuilder", "unitTests"})
   public void urlBuilderTests() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00112)